src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -24,6 +25,12 @@ public SnapshotBuilderSettings updateSnapshotBuilderSettings(
     return snapshotBuilderSettingsDao.upsertSnapshotBuilderSettingsByDataset(id, settings);
   }
 
+  public SnapshotBuilderAccessRequest createSnapshotRequest(
+      // TODO: add given request to the database, return real JobModel with the id of the new entry.
```

Comment:
Nit: This comment is no longer accurate wrt the `JobModel`. Also, could you reference the specific ticket number?